src/App.tsx

```diff
@@ -1,10 +1,64 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useCallback, useEffect, useState } from 'react';
+import CN from 'classnames';
 import { UserWarning } from './UserWarning';
+import { getTodos } from './api/todos';
+import { Todo } from './types/Todo';
+import { TodoList } from './components/TodoList/TodoList';
+import { Filter } from './components/Filter/Filter';
 
-const USER_ID = 0;
+const USER_ID = 10349;
+
+enum Options {
+  ALL = 'all',
```

Comment:
move to the separate file
```suggestion
enum Options {
  All = 'all',
```